.github/ISSUE_TEMPLATE/my-files-don-t-convert.md

```diff
@@ -0,0 +1,38 @@
+---
+name: My files don't convert
+about: Use this when you have problem converting files using echopype
+title: "[File conversion] SUSPECT_CAUSE"
+labels: my_files_don't_convert
+assignees: ''
+
+---
+
+### General description of problem
+_Required_
+
+### Computing environment
+_Required_
+- Echopype version:
+- How echopype was installed (e.g., conda or pip):
+- Operating system:
+
+### Minimum example
+_Required: A minimum code snippet that reproduces your problem._
+The following code reproduces the errors I encountered:
+```python
+CODE SNIPPET
+```
+
+### Error message printouts
+_Required: Copy-paste the entire error messages you encounter here._
+Below is the error messages I received when running the above code:
+```python
+ERROR MESSAGES
+```
+
+### Provide an example file
+_Required: An example file that allow reproducing the problem._
+Here is a link to an example file: SOME_ONLINE_LOCATION
```

Comment:
Should we place limits on how big the file can be? Say, < 200 MB?

Reply:
Yeah that will be good. I'll add that.